src/olympia/accounts/views.py

```diff
@@ -324,6 +331,8 @@ def post(self, request):
             'user_id': user.pk,
             'username': user.username,
             'email': user.email,
+            'display_name': user.display_name,
+            'groups': list((g.pk, g.name, g.rules) for g in user.groups.all()),
```

Comment:
This seems like a lot more info than we'll need but I guess there's no harm in it.